src/components/EmailGraphs.vue

```diff
@@ -1,116 +0,0 @@
-<template>
-  <div class="graphs">
-    <div class="graph">
-      <router-link class="header" to="/graphs/frommetogmail">
-        <h3>{{ fmtgLabel }}</h3>
-      </router-link>
-      <LineChart
-        :title="fmtgLabel"
-        :times="times"
-        :fromMeToGmail="fromMeToGmail"
-        :width="width"
-        :height="height"
-      />
-    </div>
-    <div class="graph">
-      <router-link class="header" to="/graphs/frommetonongmail">
-        <h3>{{ fmtngLabel }}</h3>
-      </router-link>
-      <LineChart
-        :title="fmtngLabel"
-        :times="times"
-        :fromMeToGmail="fromMeToNonGmail"
-        :width="width"
-        :height="height"
-      />
-    </div>
-    <div class="graph">
-      <router-link class="header" to="/graphs/tomefromgmail">
-        <h3>{{ tmfgLabel }}</h3> </router-link
-      ><LineChart
-        :title="tmfgLabel"
-        :times="times"
-        :fromMeToGmail="toMeFromGmail"
-        :width="width"
-        :height="height"
-      />
-    </div>
-    <div class="graph">
-      <router-link class="header" to="/graphs/tomefromnongmail">
-        <h3>{{ tmfngLabel }}</h3>
-      </router-link>
-      <LineChart
-        :title="tmfngLabel"
-        :times="times"
-        :fromMeToGmail="toMeFromNonGmail"
-        :width="width"
-        :height="height"
-      />
-    </div>
-  </div>
-</template>
-
-<script>
-import { ref } from 'vue'
-import { useStore } from 'vuex'
-import LineChart from './LineChart.vue'
-import { rangeScan } from '../fetchStats.js'
-
-export default {
-  name: 'EmailGraphs',
-  components: {
-    LineChart
-  },
-  async setup() {
-    // Setup and fetch
-    const store = useStore()
-    const start = Math.floor(
```

Comment:
Awesome, I have prior experience with this library. It's a great one, although I should warn that the maintainers have labeled it as a legacy project as of September 2020. [Read about their statement here](https://momentjs.com/docs/#/-project-status/).

Reply:
That just made my code a whole lot cleaner LOL. Thanks again for the tip.

Reply:
Yeah it's a big rip. Unlikely that we'll be moving off moment since too much of our code uses it. Heard `luxon` is the replacement but I've never used it myself